README.md

```diff
@@ -14,6 +18,18 @@ docker run --net=host -e 'ACCESS_KEY=accessKey' -e 'SECRET_KEY=secretkey' -e 'EN
 node crrExistingObjects.js testbucket1,testbucket2
 ```
 
+## Trigger CRR on *all* objects of a bucket, including those that have
```

Comment:
Not sure if it's intentional but the heading appears to be either cut off or included in header by accident.
`Trigger CRR on *all* objects of a bucket` makes sense as header.